src/main/java/bio/terra/workspace/service/workspace/flight/GoogleBucketFlightMapKeys.java

```diff
@@ -0,0 +1,33 @@
+package bio.terra.workspace.service.workspace.flight;
+
+import bio.terra.workspace.generated.model.GoogleBucketCreationParameters;
+import bio.terra.workspace.generated.model.GoogleBucketDefaultStorageClass;
+import bio.terra.workspace.generated.model.GoogleBucketLifecycle;
+
+/**
+ * Keys into the flight parameter map along with their types. Corresponds to
+ * GoogleBucketCreationParameters
+ */
+public enum GoogleBucketFlightMapKeys {
+  NAME("name", String.class),
```

Comment:
This is an interesting approach. Should we put this into the flight class itself?

Reply:
Potentially.